src/RequestHelpers.ts

```diff
@@ -0,0 +1,29 @@
+import {RibbitRequestData, RibbitRequestParams} from "./Ribbit";
```

Comment:
Can we use single quotes for these imports? Linter is mad.

Reply:
Good catch! Singlequoted